vnext/ReactNativeWindows.props

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Dont we already have a props file that everyone is supposed to import?

Reply:
I didn't see that. also bundle.props is a little tricky and it should be imported at the end of the project file.